aten/src/ATen/native/cpu/ComplexHelperKernel.cpp

```diff
@@ -0,0 +1,32 @@
+#include <ATen/Dispatch.h>
+#include <ATen/native/ComplexHelper.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
+
+namespace at { namespace native {
+namespace {
+
+void complex_kernel(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES(iter.input_dtype(), "complex_cpu", [&]() {
```

Comment:
We should only dispatch this for `torch.float` and `torch.double` dtypes.

Reply:
I think AT_DISPATCH_FLOATING_TYPES only does those two (it doesn't include torch.half).